tests/net/socket/tls/testcase.yaml

```diff
@@ -0,0 +1,7 @@
+common:
+  tags: net socket tls
+  # these platforms exhibited out of memory conditions
```

Comment:
If we have min_ram setting below, is it really necessary to blacklist certain platforms?

Reply:
I'll give it a shot without shortly.